src/Stratis.Bitcoin.Features.Miner/MiningFeature.cs

```diff
@@ -189,8 +189,8 @@ public override void ValidateDependencies(IFullNodeServiceProvider services)
             if (this.minerSettings.Mine || this.minerSettings.Stake)
             {
                 services.Features.EnsureFeature<BlockStoreFeature>();
-                var blockSettings = services.ServiceProvider.GetService<StoreSettings>();
-                if (blockSettings.Prune)
+                var storeSettings = services.ServiceProvider.GetService<StoreSettings>();
+                if (storeSettings.Prune != 0)
                     throw new ConfigurationException("BlockStore prune mode is incompatible with mining and staking.");
```

Comment:
But staking or mining should be enabled in pruned mode. 
I see no reason why it can't be enabled

Reply:
We can't enable staking or mining here as this was not how Breeze was before.

Reply:
Yeah. But prune mode is not for breeze only. FN will have pruning option too. 

Breeze is basically a FN with pruning enabled.